muxpi/nuc/flash.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+# This script downloads, flashes the image and tests it on NUC. 
+# This script must be run on muxpi.
+#
+# Usage: 
+# $ ./flash.sh [DUP IP][.tar image][neptune/no parameters for minimal]
+# The default image to use is the minimal image. If you want to run the neptune image
+# use the option 'neptune'.
+
+
+set -e
+
+DUT_IP="$1"
+VARIANT="$2"
+NEPTUNE_IMAGE="$3"
+
+HOME="/home/muxpi"
+IMAGES="${HOME}/images"
+
+if [ "$VARIANT" == "" ]; then
+   echo "VARIANT variable is empty"
```

Comment:
Use this or similar instead,
"Unspecified variant"

Don't use code variable names for error messages

Reply:
I changed to "Specify the name of image"